### DISCUSSION TITLE
댓글, 대댓글 api 연결 관련 논의

### DISCUSSION THREAD

**Discussion by JangGusWjd** [MajorIn-ABO/MajorIn_FE#69] [General]
대댓글 같은 경우에 지금 예를 들어서 1번 스터디 글에 대해서 댓글이랑 대댓글을 불러올 때
1. `/studys/posts/comments-by-postid/${parsedStudyId}/`여기서 1번글에 대한 전체 댓글 불러옴
2. `/studys/posts/comments-by-parent/${parent_comment}/` 여기서 대댓글 불러옴

이런식인데 그러면 댓글에서만 api를 두개를 불러와야 되고, 1번 글에 대해서 2개의 불러온 댓글,대댓글 api에 대해 대댓글이 어떤 댓글에 대한 대댓글인지 비교까지 해야되는 문제가 있어서

`/studys/posts/comments-by-postid/${parsedStudyId}/`이 주소에서 1번 글에 대한 댓글을 불러온다고 하면
```
[
    {
        "id": 1,
        "user_id": 53,
        "post_id": 1,
        "parent_comment": null,
        "contents": "코딩 열심히 하세요",
        "comment_date": "2024-06-01T16:08:20.327390",
        "school_name": "단국대학교",
        "major_name": "소프트웨어학과",
        "admission_date": 2020,
        // 대댓글 배열 추가
       "comments":  [
            {
                commentId: 1,
                content: "대댓글1",
              },
            {
                commentId: 2,
                content: "대댓글 2",
              },
        ]
    }
]
```

이런식으로 한번에 대댓글까지 불러오게 하는건 어때?
대댓글 post할 때 주소만 다르게 하고 댓글, 대댓글은 한번에 불러오게 하는걸로!

**Comment by cho-ja-young** [comment]
오키오키 그렇게 구조 바꿔둘게!
